### PR TITLE
Add a Thank You

### PR DESCRIPTION
**Pull request by akshit15013**
Description : Add a Thank You in Estonian Language
Link to the website : https://akshit15013.github.io/gci18.fossasia5.org/
Issue No. : #341

### CODE REVIEW

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE , CAN YOU PLEASE MERGE IT?

File: js/thankyou.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 'আপনার অবদানের জন্য আপনাকে ধন্যবাদ!',
 'Merci pour vos contributions!',
 'Kea le leboha ka menehelo ea hau!',
+"Tänan teid panuse eest!"
```
Comment (liveHarshit) [inline js/thankyou.js]: Write in the same format, it's not working.

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE , CAN YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
@sk9331657 Can you please merge it?

**Pull request merged**